### PR TITLE
Welcome bot for contributors

### PR DESCRIPTION
**Pull request by VenkyOS**
Signed-off-by: Venkataraman Padmanabhan <venkataraman.padmanabhan@seagate.com>

A GitHub bot that welcomes new users based off maintainer defined comments that must be located in .github/config.yml

It combines 3 plugins which has already being tested in CORTX repo,

- new-issue-welcome

- new-pr-welcome

- first-pr-merge

### CODE REVIEW

**Review by huanghua78 — APPROVED** [review]

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+requestInfoLabelToAdd: request-more-info
+  
+# Configuration for sentiment-bot
+# *Required* toxicity threshold between 0 and .99 with the higher numbers being the most toxic
+# Anything higher than this threshold will be marked as toxic and commented on
+sentimentBotToxicityThreshold: .7
+
+# *Required* Comment to reply with
+sentimentBotReplyComment: >
+  Please be sure to review the code of conduct and be respectful of other users. cc/ @johnbent
```
Comment (mukul-seagate11) [inline .github/config.yml:31]: please update & correct,

Please be sure to review the code of conduct and be respectful of other users. cc/ @@novium258

**Review by mukul-seagate11 — APPROVED** [review]
please update & correct,

Please be sure to review the code of conduct and be respectful of other users. cc/ @@novium258

**Comment by mukul-seagate11** [comment]
@motr, please check with RE team to resolve **mini-provisioning** issue so that the PR can be merged

**Comment by huanghua78** [comment]
> @motr, please check with RE team to resolve **mini-provisioning** issue so that the PR can be merged

What is this related to **mini-provisioning**?

**Comment by mukul-seagate11** [comment]
@shailesh-vaidya, I hope this process is not enabled yet for components?

**Comment by shailesh-vaidya** [comment]
> @shailesh-vaidya, I hope this process is not enabled & approved yet for components?

@mukul-seagate11  - Do you mean `mini-provisioner` .  It's not mandatory yet. If you see that check is not mandatory for PR merge.

**Comment by mukul-seagate11** [comment]
@huanghua78, you can merge the PR & ignore **mini-provisioning** for now which will actually rolled-out or communicated later once approved from ARC

**Pull request merged**